appdatabase/migrations/sql/1709795716_migration_order_fix.up.sql

```diff
@@ -0,0 +1,101 @@
+)
+SELECT
+    address,
+    chaos_mode,
+    currency,
+    current_network,
+    custom_bootnodes,
+    custom_bootnodes_enabled,
+    dapps_address,
+    eip1581_address,
+    fleet,
+    hide_home_tooltip,
+    installation_id,
+    key_uid,
+    keycard_instance_uid,
+    keycard_paired_on,
+    keycard_pairing,
+    last_updated,
+    log_level,
+    mnemonic,
+    name,
+    networks,
+    node_config,
+    notifications_enabled,
+    photo_path,
+    pinned_mailservers,
+    preferred_name,
+    preview_privacy,
+    public_key,
+    remember_syncing_choice,
+    signing_phrase,
+    stickers_packs_installed,
+    stickers_recent_stickers,
+    syncing_on_mobile_network,
+    synthetic_id,
+    usernames,
+    wallet_root_address,
+    wallet_set_up_passed,
+    wallet_visible_tokens,
+    stickers_packs_pending,
+    waku_enabled,
+    waku_bloom_filter_mode,
+    appearance,
+    remote_push_notifications_enabled,
+    send_push_notifications,
+    push_notifications_server_enabled,
+    push_notifications_from_contacts_only,
+    push_notifications_block_mentions,
+    webview_allow_permission_requests,
+    use_mailservers,
+    link_preview_request_enabled,
+    link_previews_enabled_sites,
+    profile_pictures_visibility,
+    anon_metrics_should_send,
+    messages_from_contacts_only,
+    default_sync_period,
+    current_user_status,
+    send_status_updates,
+    gif_recents,
+    gif_favorites,
+    opensea_enabled,
+    profile_pictures_show_to,
+    telemetry_server_url,
+    backup_enabled,
+    last_backup,
+    backup_fetched,
+    auto_message_enabled,
+    gif_api_key,
+    display_name,
+    test_networks_enabled,
+    mutual_contact_enabled,
+    bio,
+    mnemonic_removed,
+    latest_derived_path,
+    device_name,
+    wallet_accounts_position_change_clock,
+    profile_migration_needed,
+    is_sepolia_enabled,
+    url_unfurling_mode,
+    omit_transfers_history_scan,
+    wallet_show_community_asset_when_sending_tokens,
+    wallet_display_assets_below_balance,
+    wallet_display_assets_below_balance_threshold,
+    wallet_collectible_preferences_change_clock,
+    wallet_collectible_preferences_group_by_collection,
+    wallet_collectible_preferences_group_by_community,
+    peer_syncing_enabled,
```

Comment:
This will fail on devices that haven't migrated, as it's not in the settings table, so `peer_syncing_enabled` should not be set and defaulted to false